cishouseholds/pipeline/run.py

```diff
@@ -45,25 +48,29 @@ def run_from_config():
     add_run_status(run_id, "finished")
 
 
-def run_pipeline_stages(pipeline_stage_list: list, run_id: int):
+def run_pipeline_stages(pipeline_stage_list: list, run_id: int, retry_count: int, retry_wait_time: int):
     """Run each stage of the pipeline. Catches, prints and logs any errors, but continues the pipeline run."""
     number_of_stages = len(pipeline_stage_list)
     max_digits = len(str(number_of_stages))
     pipeline_error_count = 0
     for n, stage_config in enumerate(pipeline_stage_list):
         stage_start = datetime.now()
```

Comment:
I think this should be in the loop, otherwise the "completed in" time will be cumulative for each retry

Reply:
i thought that would be  how it should be to show that the stage required that total time to complete as each retry accounts for that same end output